src/python_testing/TC_OpstateCommon.py

```diff
@@ -1113,7 +1113,7 @@ async def TEST_TC_OPSTATE_BASE_2_5(self, endpoint=1):
             # In CI environment, the STOP coommand (step 8) already resets the variables. Only ask for
             # reboot outside CI environment.
             if not self.is_ci:
-                input("Press Enter when done.\n")
+                self.wait_for_user_input(prompt_msg="Press Enter when ready.\n")
```

Comment:
same

Reply:
This one is more straightforward than the comment above since this appears to be tied to a specific step (11 - Restart DUT). I added this change (improving the string) to my local repo but haven't committed it yet, since it is in the same file as the comment above.